mcp-annotations/src/test/java/org/springaicommunity/mcp/method/tool/AsyncMcpToolMethodCallbackTests.java

```diff
@@ -120,7 +121,8 @@ public Mono<String> enumMonoTool(TestEnum enumValue) {
 
 		@McpTool(name = "primitive-types-mono-tool", description = "Mono tool with primitive types")
 		public Mono<String> primitiveTypesMonoTool(boolean flag, byte b, short s, int i, long l, float f, double d) {
-			return Mono.just(String.format("Primitives: %b, %d, %d, %d, %d, %.1f, %.1f", flag, b, s, i, l, f, d));
+			return Mono
+				.just(String.format(Locale.US, "Primitives: %b, %d, %d, %d, %d, %.1f, %.1f", flag, b, s, i, l, f, d));
```

Comment:
I got a failure on one of the unit tests since decimal represented as , at my machine. Thus, explicitly use Locale.US to handle it.